misk/src/test/kotlin/misk/config/MiskConfigTest.kt

```diff
@@ -112,4 +112,22 @@ class MiskConfigTest {
     assertThat(filesInDir[0]).endsWith("/overrides/override-test-app1.yaml")
     assertThat(filesInDir[1]).endsWith("/overrides/override-test-app2.yaml")
   }
+
+  @Test
+  fun handlesDuplicateNamedExternalFiles() {
+    val overrides = listOf(
+      MiskConfigTest::class.java.getResource("/overrides/override-test-app1.yaml"),
+      MiskConfigTest::class.java.getResource("/additional_overrides/override-test-app1.yaml"))
+      .map { File(it.file) }
+
+    val config = MiskConfig.load<TestConfig>("test_app", defaultEnv, overrides)
+    assertThat(config.consumer_a).isEqualTo(ConsumerConfig(14, 1))
+    assertThat(config.consumer_b).isEqualTo(ConsumerConfig(34, 79))
```

Comment:
oh we should make sure these assertions check values from both resources

Reply:
Just confirmed, they do